azure-ts-dynamicresource/index.ts

```diff
@@ -0,0 +1,86 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+import { CDNCustomDomainResource } from "./cdnCustomDomain";
+
+const location = "West US";
+
+/** The custom domain host of the CDN endpoint. */
+export let cdnCustomDomainResource: CDNCustomDomainResource | undefined;
+
+// Create an Azure Resource Group
+const resourceGroup = new azure.core.ResourceGroup("resourceGroup", {
+    location: location,
+});
+
+// Create an Azure resource (Storage Account) so we can point the CDN endpoint to it.
+const storageAccount = new azure.storage.Account("storage-account", {
```

Comment:
Storage account names can't have dashes in them

Reply:
Ah yes. This is the result of me trying to change the names I used in my personal project to something generic. I'll change this.